modules/installation-nutanix-failure-domains-req.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/nutanix-failure-domains.adoc
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-nutanix-failure-domains-req_{context}"]
+= Failure domain requirements
+
+When planning to use failure domains, consider the following requirements:
+
+* All Prism Element instances must be managed by the same instance of Prism Central. A deployment that is comprised of multiple Prism Central instances is not supported.
```

Comment:
The Failure Domain Prism Element clusters would need to be in the same L2 subnet.

Reply:
Can you clarify this comment? Does this mean that the machines that make up the Nutanix environment (for example, bare metal servers) need to be in the same L2 subnet? Or is there a configuration option when creating Prism Element clusters that would satisfy this requirement? Thank you

Reply:
The machines that make up the Nutanix environment (for example, bare metal servers) need to be in the same L2 subnet.

Reply:
```suggestion
* All Prism Element instances must be managed by the same instance of Prism Central. A deployment that is comprised of multiple Prism Central instances is not supported.
* The machines that make up the Prism Element clusters must reside on the same Ethernet network for failure domains to be able to communicate with each other.
```
@yanhua121 Is this an accurate summary of your feedback? Thanks

Reply:
@bscott-rh Yes, I think so.